config/routes.rb

```diff
@@ -7,7 +7,14 @@
   # root "articles#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create]
   end
+
+  
+  # Add comment to a post
+  post 'users/:user_id/posts/:id/', to: 'posts#comment_post', as: 'comment_post'
+
+  # Like a post
+  post '/users/:user_id/posts/:id/like', to: 'posts#like_post', as: 'likes'
```

Comment:
- [ ] Add `like` controller so that a user can be able to like a post

Reply:
Hello @DeliceLydia , i have already implemented the controllers, its routes and view layouts for adding a new post, please kindly checkout it out at :
http://localhost:3000/users/1/posts/new 

as shown in the picture below
![Screenshot (120)](https://user-images.githubusercontent.com/98436409/201494812-8f39aef3-ca89-4bfd-8579-045387926aaf.png)

and also controllers, its routes and forms for adding a comment and like a post as well
at   http://localhost:3000/users/1/posts/1
as shown as well  in the picture below
![Screenshot (119)](https://user-images.githubusercontent.com/98436409/201494959-c29cfb2a-57fb-486c-9b6f-94b4703c7988.png)